docs/docs/headless-cms.md

```diff
@@ -21,33 +21,33 @@ The guides in this section will walk through the process of setting up content s
-| [AgilityCMS](https://agilitycms.com/)         | [guide](/docs/sourcing-from-agilitycms/)                                         | [docs](/packages/@agility/gatsby-source-agilitycms/) | [starter](/starters/agility/agility-gatsby-starter/)               |
-| [Forestry](https://forestry.io/)              | [guide](/docs/sourcing-from-forestry/)                                           |                                                      |                                                                    |
-| [Gentics Mesh](https://getmesh.io)            | [guide](/docs/sourcing-from-gentics-mesh)                                        |                                                      |                                                                    |
-| [Seams-CMS](https://seams-cms.com/)           | [guide](/docs/sourcing-from-seams-cms)                                           |                                                      |                                                                    |
-| [Builder.io](https://www.builder.io/)         | [guide](/docs/sourcing-from-builder-io/)                                         | [docs](/packages/@builder.io/gatsby/)                | [starter](https://github.com/BuilderIO/gatsby-starter-builder)     |
-| [Flotiq](https://flotiq.com/)                 | [guide](/docs/sourcing-from-flotiq/)                                             | [docs](/packages/gatsby-source-flotiq)               | [starter](https://github.com/flotiq/gatsby-starter-blog)           |
+| CMS                                           | Guides                                                                           | Plugin Docs                                            | Official Starter                                                   |
+| --------------------------------------------- | -------------------------------------------------------------------------------- | ------------------------------------------------------ | ------------------------------------------------------------------ |
+| [Contentful](https://www.contentful.com/)     | [guide](/docs/sourcing-from-contentful/)                                         | [docs](/packages/gatsby-source-contentful)             | [starter](/starters/contentful/starter-gatsby-blog/)               |
+| [NetlifyCMS](https://www.netlifycms.org/)     | [guide](/docs/sourcing-from-netlify-cms/)                                        | [docs](/packages/gatsby-plugin-netlify-cms)            | [starter](/starters/netlify-templates/gatsby-starter-netlify-cms/) |
+| [WordPress](https://www.wordpress.com/)       | [guide](/docs/sourcing-from-wordpress/)                                          | [docs](/packages/gatsby-source-wordpress-experimental) |                                                                    |
```

Comment:
The guide here looks like it's a 404. Maybe it should link here? https://github.com/gatsbyjs/gatsby-source-wordpress-experimental/blob/master/docs/getting-started.md

Or maybe this link will exist once this pr is merged?

Reply:
It should already exist cause this is an update, not a new guide

Reply:
it works...https://www.gatsbyjs.org/docs/sourcing-from-wordpress/

Reply:
ah, my bad, I went to the rich view and clicked the link. it's a relative link 🤦

Reply:
Thanks, please merge, I can't on this repo.

Reply:
It should merge once all the tests pass